go/chat/server.go

```diff
@@ -1720,6 +1720,29 @@ func (h *Server) DeleteConversationLocal(ctx context.Context, arg chat1.DeleteCo
 	return res, nil
 }
 
+func (h *Server) RemoveFromConversationLocal(ctx context.Context, arg chat1.RemoveFromConversationLocalArg) (res chat1.RemoveFromConversationLocalRes, err error) {
+	var identBreaks []keybase1.TLFIdentifyFailure
+	ctx = globals.ChatCtx(ctx, h.G(), keybase1.TLFIdentifyBehavior_CHAT_GUI,
+		&identBreaks, h.identNotifier)
+	defer h.Trace(ctx, &err, fmt.Sprintf("RemoveFromConversation(%s)", arg.ConvID))()
+	defer func() { err = h.handleOfflineError(ctx, err, &res) }()
```

Comment:
dont' think we should squash these, if the user is offline this should just fail

Reply:
That makes a lot of sense to me, although most of the other functions do have an offline case (ie. `leaveConversation`: https://github.com/keybase/client/pull/23979/files/b60247997717484f2efdbbda6a14875c7bf2a53a#diff-81838f60f2a58408d6075918338196c5R1637-R1659)

I'm happy to get rid of it though, I don't really know what it would do if you were offline 😅